crates/sui-types/src/collection_types.rs

```diff
@@ -22,3 +24,68 @@ pub struct Entry<K, V> {
 pub struct VecSet<T> {
     contents: Vec<T>,
 }
+
+/// Rust version of the Move std::option::Option type.
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq, JsonSchema)]
+pub struct MoveOption<T> {
```

Comment:
Not sure if Option is considered a collection type but I'm fine with it staying here.

Reply:
Yeah maybe we will eventually add another file called common_move_types?